RadixWallet/Profile/Factor/FactorSourceID.swift

```diff
@@ -92,9 +92,11 @@ extension FactorSourceID.FromHash {
 	/// NEVER EVER CHANGE THIS! If you do, users apps will be unable to load the Mnemonic
 	/// from keychain!
 	public var keychainKey: String {
-		"\(kind):\(body.data.hex())"
+		"\(kind)\(Self.keychainKeySeparator)\(body.data.hex())"
 	}
 
+	static let keychainKeySeparator = ":"
```

Comment:
used by a new failable init, taking this keychainKey as input and trying to create a FactorSourceID from it.